### DISCUSSION TITLE
Apache NetBeans 24 release candidates

### DISCUSSION THREAD

**Discussion by ebarboni** [apache/netbeans#7898] [General]
Release candidates for the next release of Apache NetBeans will be available weekly until release. Your help in testing, reporting and verifying issues would be highly appreciated by the whole Apache NetBeans community.

Changes from NetBeans 23 can be seen at https://github.com/apache/netbeans/releases/tag/24-rc1

## Binary / source zip

Binary and source zips are available from -

https://nightlies.apache.org/netbeans/candidate/netbeans/

## Installers

Unsigned Windows, Linux DEB and Linux RPM installers are available from -

https://nightlies.apache.org/netbeans/candidate/installers/

NB. We are working on making other installers available for release candidates from the same location.

## VSCode plugin

The VSCode plugin is available from - 

https://nightlies.apache.org/netbeans/candidate/vsix/

## IMPORTANT

These release candidates are pre-release software, and not official releases of Apache NetBeans.

**Comment by mbien** [comment]
NetBeans 24 is now available for download https://netbeans.apache.org/download/nb24/, closing
